onadata/libs/permissions.py

```diff
@@ -251,6 +261,33 @@ class DataEntryRole(Role):
     }
 
 
+class EditorNoDownload(Role):
+    """
+    Editor-Minor Role class - user can submit data, read and edit only the data
```

Comment:
Editor-Minor Role class does not match the class name, either way, I think we can omit it the ` Editor-Minor Role class -` prefix altogether